### ISSUE TITLE
Elixir 1.2 - Cannot Detect SDK Version

### ISSUE THREAD

**Issue by swennemans** [KronicDeth/intellij-elixir#232]
I've updated from Elixir 1.1.1 to 1.2 and cannot set the Project SDK anymore. 

I've tried installing Elixir with: 
* Homebrew
* MacPorts
* Git Clone

All result in the same error: 
![image](https://cloud.githubusercontent.com/assets/7531154/12140244/a385a1b0-b466-11e5-8096-2ee652a1f0eb.png)

If you need any more info please let me know :)

**Comment by swennemans** [comment]
BTW: If someones has the same problem, you can always install Elixir 1.1.1 with homebrew with: 
`brew install https://raw.githubusercontent.com/Homebrew/homebrew/373f8bfba76070d701a9796be11e8f280e465aa7/Library/Formula/elixir.rb`

**Comment by gmack** [comment]
Same issue for me. I get the error on both RubyMine or IntelliJ. FWIW, I build my own Elixir 1.2 from a git clone.

**Comment by net** [comment]
+1

**Comment by se77en** [comment]
Same issue for me. I upgrade elixir use Homebrew.

**Comment by pcmarks** [comment]
Same problem: "Probably SDK  is corrupt" for Elixir 1.2 on WIndows 7 using IntelliJ 14

**Comment by bitgamma** [comment]
The output of the `elixir -v` command is

```
Erlang/OTP 18 [erts-7.2.1] [source] [64-bit] [smp:8:8] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]

Elixir 1.2.0
```

I think in earlier versions of Elixir only the last line was present in the output and parsing of the version fails (at least that was my impression by a quick analysis of the code). I will try to submit a PR soon making the parsing more flexible if it's OK for @KronicDeth.

**Comment by KronicDeth** [comment]
That works for me @bitgamma.  Just make sure your PR works for both < 1.2 and >= 1.2 for those that haven't upgraded yet too.

**Comment by robertsosinski** [comment]
+1

**Comment by KronicDeth** [comment]
Reproduced bug on OSX prior to testing #233.

**Comment by daemonfire300** [comment]
Is there going to be a 2.0.1 Release for this, or is there are possibility to build the Plugin myself so that I can use Elixir 1.2.0 right now?

**Comment by KronicDeth** [comment]
@daemonfire300 no 2.0.1, I [just released 2.1.0](https://twitter.com/KronicDeth/status/686274404708298752) instead as it contains more Elixir 1.2 compatibility fixes than the SDK support (the build actually tests against 1.1.1 and 1.2.0 now on travis-ci and the lexer/parser supports `\u` in strings/char lists.)
